### ISSUE TITLE
List items have nested paragraph tags by default

### ISSUE THREAD

**Issue by alexroper** [verbb/vizy#153]
**Description**

When creating ordered and unordered lists, list items have nested paragraph tags by default.

```
<ul>
    <li>
        <p>Test</p>
    </li>
</ul>
```

You can see this behavior by revealing the HTML code on the Vizy field itself, or by rendering the field on the front end like this:

```
{{ entry.text.renderHtml() }}
```

The expected behavior is that paragraph tags are converted into `<li>` tags instead of wrapping them when the list button is clicked. Expected output would be:

```
<ul>
    <li>Test</li>
</ul>
```

**Steps to reproduce**
1. Create an unordered list in a Vizy field.
2. There is no step 2.

**Additional info**
- Plugin version: 2.0.2
- Craft version: Craft Pro 4.0.4
- Multi-site: No

**Additional context**

**Comment by alexroper** [comment]
I think this is a default behavior inherited from ProseMirror. It happens in their basic example:

https://prosemirror.net/examples/basic/

I'm not sure if it's possible to override this. It's unfortunate that this is the default behavior. Neither Redactor nor CKEditor behave this way. It's perplexing, but I'm guessing this is an intentional choice by ProseMirror's authors.

**Comment by engram-design** [comment]
Yeah, it's a known issue coming from ProseMirror I'm afraid. I've looked up ways to handle it, and I think it comes down to how new items are handled, and marks like Bold, Italic, etc are handled.

But there are some potential [changes](https://discuss.prosemirror.net/t/removing-the-default-paragraph-p-inside-a-list-item-li/2745/14) that I'll need to test out.

**Comment by engram-design** [comment]
Fixed in [2.1.0](https://github.com/verbb/vizy/releases/tag/2.1.0)
